mitmproxy/tools/console/statusbar.py

```diff
@@ -60,6 +60,33 @@ def cb(*args):
     def prep_prompt(self, p):
         return p.strip() + ": "
 
+    def prep_message(self, msg):
```

Comment:
`prep_message` isn't very telling, maybe `shorten_message`? We can add a docstring to the function that explains why we do this, e.g. `"""Shorten message so that it fits into a single line in the statusbar."""`